test/functional/sync_plan/info_test.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/sync_plan'
+require_relative 'sync_plan_helpers'
+require_relative '../organization/organization_helpers'
+module HammerCLIKatello
+  describe SyncPlan::InfoCommand do
+    include OrganizationHelpers
+    include SyncPlanHelpers
+
+    it 'requires ID' do
+      api_expects(:sync_plans, :show).with_params('id' => 1)
+
+      run_cmd(%w(sync-plan info --id 1))
+    end
+
+    it 'requires organization id and sync plan name' do
+      api_expects(:sync_plans, :index).with_params('organization_id' => 1, 'name' => 'syncplan1')
```

Comment:
You should still test that `:sync_plans, :show` gets called with the correct params.